### PR TITLE
Add k8s definition for the cote workshop example.

### PR DESCRIPTION
**Pull request by drubin**
Usage `kubectl apply -f docker-k8s.yml`

I wanted to refactor all of the different setups into different folders each with their own readme/getting started as it seemed useful but I didn't want to complicate the different change sets.

### CODE REVIEW

File: docker-k8s.yml [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    matchLabels:
+      name: end-user
+  template:
+    metadata:
+      labels:
+        name: end-user
+    spec:
+      containers:
+      - image: dashersw/cote-workshop
+        name: admin
```
Comment (dashersw) [inline docker-k8s.yml]: The name is c/p'd here, I guess.

File: docker-k8s.yml [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+    spec:
+      containers:
+      - image: dashersw/cote-workshop
+        name: init-db
+        command: ["node", "init-db.js"]
+        env:
+          - name: PG
+            value: "true"
+          - name: COTE_MULTICAST_ADDRESS
+            value: '239.1.11.131'
```
Comment (dashersw) [inline docker-k8s.yml]: should this be double-quotes?

File: docker-k8s.yml [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+      - image: dashersw/cote-workshop
+        name: admin
+        command: ["node", "end-user/server.js"]
+        ports:
+        - containerPort: 5001
+        env:
+          - name: PG
+            value: "true"
+          - name: COTE_MULTICAST_ADDRESS
+            value: '239.1.11.131'
```
Comment (dashersw) [inline docker-k8s.yml]: is there something like a variable in k8s templates? there's a lot of repetition in the file, including but not limited to the multicast address.

File: docker-k8s.yml [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+    matchLabels:
+      name: product-service
+  template:
+    metadata:
+      labels:
+        name: product-service
+    spec:
+      containers:
+      - image: dashersw/cote-workshop
+        name: admin
```
Comment (dashersw) [inline docker-k8s.yml]: here again.

File: docker-k8s.yml (hunk 3 above)
Comment (drubin) [reply docker-k8s.yml]: Yes k8s support configmaps which can share configs such as these between deployments.  I didn't include them as I tried to limit things we were using. 

As for the duplication of these services we could use yaml achnors or standard templates which most people do anyway.

**Comment by LeoVS09** [comment]
Hi, I trying use cote in k8s. And this example can be helpful, but can you explain what is address used for COTE_MULTICAST_ADDRESS? It's host addpress or some pod?

**Comment by drubin** [comment]
@LeoVS09 Actually it is just a (semi-random) address that each pod /can/ get too. This is how MULTICAST works.

But this only works if your underlying network CNI plugin supports MULTICAST, which currently is the only weavenet OR you run a single node cluster.

For every thing else right now the best bet is to use the redis service discovery.

**Pull request closed without merging**